src/Calendar.js

```diff
@@ -518,15 +518,7 @@ let Calendar = React.createClass({
     onNavigate(date, view)
 
     if (action === navigate.DATE)
-      this._viewNavigate(date)
-  },
-
-  _viewNavigate(nextDate) {
-    let { view, date, culture } = this.props;
-
-    if (dates.eq(date, nextDate, view, localizer.startOfWeek(culture))) {
```

Comment:
@tobiasandersen I remember why this check was here. If you try to navigate to an arbitrary date from the month view you can't now without it also switching to the day view